package 异常;

public class ExceptionTest1 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		Integer i1 = null;
		try
		{
			int i2 = i1;
		}
		catch(NullPointerException e)
		{
			System.out.println("i不能为null")
		}
	}

}
